src/main/java/DeadlineTasks.java

```diff
@@ -8,6 +8,7 @@ public DeadlineTasks(String input, String deadline){
 
     @Override
     public String getName() {
```

Comment:
I like how the printing of the task details are extracted into methods within classes such that it can always be easily called.